msvc/config-msvc.mak.in

```diff
@@ -81,7 +81,7 @@ PSL_DLL = vs$(VSVER)\$(CFG)\$(PLAT)\$(PSL_DLL_PREFIX)psl$(PSL_DLL_SUFFIX).dll
 
 # ENABLE_RUNTIME_ICU and ENABLE_BUILTIN_ICU are currently dummy arguments to NMake
```

Comment:
Does ENABLE_BUILTIN_ICU still make sense in this comment ?

Reply:
In fact, none of them exist in the original commit.

If the comment was about the `ENABLED_RUNTIME` and `ENABLED_BUILTIN` below, which are not used for the build (only for the `build-info` target), then I can update it to `# ENABLED_RUNTIME  and ENABLE_BUILTIN are currently dummy arguments to NMake`.

Reply:
Sounds good 👍